src/main-client.ts

```diff
@@ -807,7 +808,11 @@ export class MainClient extends BaseRestClient {
 
   // TODO - https://binance-docs.github.io/apidocs/spot/en/#get-isolated-margin-transfer-history-user_data
 
-  // TODO - https://binance-docs.github.io/apidocs/spot/en/#query-isolated-margin-account-info-user_data
+  queryIsolatedMarginAccountInfo(params?: {
```

Comment:
```suggestion
  getIsolatedMarginAccountInfo(params?: {
```

Any endpoint that fetches info, I've been prefixing with `get`